core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -92,8 +93,9 @@ public GrpcBlockingStream(Function<StreamObserver<ResT>, StreamObserver<ReqT>> r
    */
   public void send(ReqT request, long timeoutMs) throws IOException {
     if (mClosed || mCanceled || mClosedFromRemote) {
-      throw new CancelledException(formatErrorMessage(
-          "Failed to send request %s: stream is already closed or canceled.", request));
+      throw new CancelledException(
+          formatErrorMessage("Failed to send request %s: stream is already closed or canceled.",
+              LogUtils.truncateMessageLines(request)));
```

Comment:
Well, the main problem is that we technically cannot assume what type the request is. It can be any type, so we cannot assume it is a proto, or even which proto it is.

I wanted to avoid adding `instanceof` calls for various proto types, and so the current mechanism is to just truncate long lines.

Reply:
fair enough